src/resources/css/themes/Rounded/BaseTrack.css

```diff
@@ -96,11 +96,12 @@ BaseTrackView
    font-size: 8px; /*db value */
 }
 
-BaseTrackView[unlighted="true"]             /* track when is not xmiting */
+BaseTrackView[unlighted="true"],                            /* the track */
+TrackGroupView[unlighted="true"] > #topPanel                /* the track group top panel: used to show user name */
```

Comment:
> This is correct? Is your intention?

Not at all. Maybe this was a bad solution. I tested some other possibilities, but none worked. I just copied the code from Flat theme and added the gradient for Rounded, but this is obviously changing other things.

The original fix should have been to change only the values in the code for `BaseTrackView[unlighted="true"]`, but I tried and there was no change.